lib/ansible/modules/cloud/openstack/os_network.py

```diff
@@ -170,6 +179,7 @@ def main():
     provider_network_type = module.params['provider_network_type']
     provider_segmentation_id = module.params['provider_segmentation_id']
     project = module.params.get('project')
+    port_security_enabled = module.params['port_security_enabled']
 
     shade, cloud = openstack_cloud_from_module(module, min_version='1.6.0')
```

Comment:
You need to raise this to 1.27.1 to match your commit message.

Reply:
Thanks. I missed that. Committed and pushed.